scripts/cli/handler/workspace.py

```diff
@@ -0,0 +1,222 @@
+             "connection options.")
+
+  #--- Create CodeCompass.export ---#
+  with open(join(projectdir, "CodeCompass.export"), 'w') as exp:
+    json.dump({
+      'name': str(args.export[0]),
+      'config': config_file_handler.get_attributes(args.export[0])
+    }, exp)
+
+  #--- Create tar.gz file ---#
+  try:
+    print "Creating archive..."
+    proc = subprocess.Popen(
+      ['tar', '-C', args.workdir, '-czf', args.export[1], args.export[0]])
+    proc.wait()
+    print "Done"
+  except Exception as e:
+    sys.exit("Failed to create dump package!")
+  finally:
+    #--- Delete dumps ---#
+    if os.path.exists(dump_filename):
+      os.remove(dump_filename)
+
+def __import_workspace(config_file_handler, args):
+  '''Imports a workspace (project dir, SQL database)'''
+  tgz_file = getattr(args, "import")
+  if not os.path.exists(tgz_file):
+    sys.exit(tgz_file + " does not exists!")
+
+  #--- Get workspace name from tgz ---#
+  with tarfile.open(tgz_file) as tar:
+    files = tar.getmembers()
+    if len(files) == 0:
+      sys.exit(tgz_file + " is empty!")
+    project_name = files[0].name
+    projectdir = join(args.workdir, project_name)
+
+  if config_file_handler.exists(project_name):
+    sys.exit("Workspace '"+ project_name +"' already exists! Use --delete to " \
+             "remove and then try again.")
+
+  #--- Unpack tar.gz file ---#
+  try:
+    print "Unpacking project '" + project_name + "' from " + tgz_file
+    proc = subprocess.Popen(
+      ['tar', '-C', args.workdir, '-xzf', tgz_file])
+    proc.wait()
+  except Exception as e:
+    sys.exit("Failed to unpack " + tgz_file)
+
+  #--- Read CodeCompass.export ---#
+  with open(join(projectdir, "CodeCompass.export"), 'r') as exp:
+    exportData = json.load(exp)
+
+  #--- Restore database ---#
+  print "Restoring database"
+  dump_filename = join(projectdir, 'database.dump')
+  util.pgsql_cmd(projectdir, args, 'dropdb', project_name)
+  restoreRetCode = util.pgsql_cmd(
+    projectdir,
+    args,
+    'pg_restore',
+    '-d', 'template1',
+    '-F', 'tar',
+    '-C', # Add CREATE DATABASE to the dump
+    '-O', # Do not set ownership in the dump
+    dump_filename)
+  if restoreRetCode != 0:
+    sys.exit("Failed! Please check " + join(projectdir, "dblog") + " for details.")
+  else:
+    #--- Delete dumps ---#
+    if os.path.exists(dump_filename):
+      os.remove(dump_filename)
+
+  #--- Add to workspace ---#
+  description = project_name;
+  if 'config' in exportData and 'description' in exportData['config']:
+    description = exportData['config']['description']
+
+  connArgs = copy.copy(args)
+  connArgs.name = project_name
+  connection_string = util.create_connection_string(connArgs)
+
+  config_file_handler.add(project_name)
+  config_file_handler.set_attribute(project_name, 'connection', connection_string)
+  config_file_handler.set_attribute(project_name, 'description', description)
```

Comment:
The `CodeCompass.export` file doesn't contain the attribute's values. I fixed this error. 
I also wrote the datadir to this file.